src/core/net_db.cc

```diff
@@ -102,7 +102,9 @@ void NetDb::Run() {
            last_manage_request = 0;
   while (m_IsRunning) {
     try {
-      auto msg = m_Queue.GetNextWithTimeout(15000);  // 15 sec
+      auto msg =
+	m_Queue.GetNextWithTimeout(
+	     static_cast<std::size_t>(NetDbDuration::WaitForMessageTimeout));  // 15 sec
```

Comment:
Like I mentioned [here](https://github.com/monero-project/kovri/pull/412#discussion_r85008739), please try to get your editor in check (no single or odd number of spaces).